client/src/pages/Feedback/styles.module.scss

```diff
@@ -0,0 +1,23 @@
+.title {
+  margin-top: 60px;
+}
+
+.main {
+  min-height: calc(100vh - 80px);
+  width: 100%;
+  padding: 20px;
+  text-align: center;
+  position: relative;
+  background-color: whitesmoke;
+  color: rgb(33, 33, 33);
+}
+
+.mainDark {
+  min-height: calc(100vh - 80px);
+  width: 100%;
+  padding: 20px;
+  text-align: center;
+  position: relative;
+  background-color: rgb(54, 54, 54);
+  color: whitesmoke;
+}
```

Comment:
class names main and mainDark are almost the same, please create one common class name